webpack/components/RoutedTabs/RoutedTabs.js

```diff
@@ -24,7 +24,7 @@ const RoutedTabs = ({
     if (matchedTab) {
       history.push(buildLink(matchedTab.key));
     } else {
-      history.push(buildLink(tabs[defaultTabIndex].key)); // go to first tab if no tab selected
+      history.replace(buildLink(tabs[defaultTabIndex].key)); // go to first tab if no tab selected
     }
```

Comment:
I'll remove this history.replace here and tackle it with the issue we have open around back button to keep things separate.

Reply:
It shouldn't always be `replace` anyway.  We don't want to lose the functionality of being able to move between different tabs with the Back button.